client/my-sites/stats/activity-log/index.jsx

```diff
@@ -256,27 +256,35 @@ class ActivityLog extends Component {
 			);
 		}
 
-		const logsGroupedByDay = map(
-			groupBy( logs, log =>
-				this.applySiteOffset( moment.utc( log.activityTs ) ).endOf( 'day' ).valueOf()
-			),
-			( dailyLogs, tsEndOfSiteDay ) =>
+		const logsGroupedByDay = groupBy( logs, log =>
+			this.applySiteOffset( moment.utc( log.activityTs ) ).endOf( 'day' ).valueOf()
+		);
+		const activityDays = [];
+		for (
```

Comment:
This is a pretty gnarly loop. The intention of the change here is to add days back in that don't have activities, is that right?

Reply:
Per some off-GH conversation, I just commented this for now (3c08851). We'll have an opportunity to revisit in #17135